src/RestWrite.js

```diff
@@ -567,6 +567,12 @@ RestWrite.prototype.handleInstallation = function() {
                           'must be specified in this operation');
   }
 
+  // Updating _Installation but not updating anything critical
```

Comment:
installationId could still be set on line 595 so should this check moved after that?